### ISSUE TITLE
azure_rm_adpassword add description field for application secrets

### ISSUE THREAD

**Issue by Klaas-** [ansible/ansible_collections_azure#1149]
##### SUMMARY

Hi,
I would like to add a description to the client secrets I create for azure ad applications. Currently it seems there is no module parameter to achieve this.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

azure_rm_adpassword

##### ADDITIONAL INFORMATION

```yaml
    - name: Create Secret in AAD Application
      azure.azcollection.azure_rm_adpassword:
        tenant: "{{ tenant_id }}"
        app_id: "{{ app_id }}"
        state: present
        end_date: "{{ '%Y-%m-%dT%H:%M:%S' | strftime((ansible_date_time.epoch | int) + (3*86400)) }}Z"
        value: "secret"
        description: This is an example description that is currently not supported
      register: azure_rm_adpassword_output
```

**Comment by Fred-sun** [comment]
@Klaas- Comments like #1150

**Comment by Klaas-** [comment]
Closed in favor of #1112 where the azure_rm_adpassword is being rewritten using a different API that supports descriptions
